file_snapshot.go

```diff
@@ -240,7 +240,11 @@ func (f *FileSnapshotStore) getSnapshots() ([]*fileSnapshotMeta, error) {
 		// Ignore any temporary snapshots
 		dirName := snap.Name()
 		if strings.HasSuffix(dirName, tmpSuffix) {
-			f.logger.Printf("[WARN] snapshot: Found temporary snapshot: %v", dirName)
+			f.logger.Printf("[WARN] snapshot: Found temporary snapshot: %v, deleting..", dirName)
```

Comment:
Hmm - this is good to do but it feels wrong as a side effect got a `getSnapshots()` call, since I don't expect that to write anything. It seems like we should attempt this over in the path where the snapshot failed.